.github/workflows/verify.yml

```
@@ -45,3 +42,23 @@ jobs:
           ./manifests/hivemq-platform/manifests.sh
           ./manifests/hivemq-swarm/manifests.sh
           git diff HEAD --exit-code
+
+      - name: Install Kubeconform Helm plugin
+        run: |
+          helm plugin install https://github.com/jtyr/kubeconform-helm
+
+      - name: Parse HiveMQ CRDs from OpenAPI to JSON schema
+        run: |
+          set -x
+          curl -fsSL -o openapi2jsonschema.py https://raw.githubusercontent.com/yannh/kubeconform/refs/heads/master/scripts/openapi2jsonschema.py
+          chmod +x openapi2jsonschema.py
+          export FILENAME_FORMAT='{kind}.{group}-{version}'
+          ./openapi2jsonschema.py https://raw.githubusercontent.com/hivemq/helm-charts/refs/heads/master/charts/hivemq-platform-operator/crds/hivemq-platforms.hivemq.com-v1.yml
+          ./openapi2jsonschema.py https://raw.githubusercontent.com/hivemq/helm-charts/refs/heads/master/charts/hivemq-operator/crds/hivemq-cluster.yaml
```

This will fail on branches with CRD changes. I think this must be:
```suggestion
          ./openapi2jsonschema.py charts/hivemq-platform-operator/crds/hivemq-platforms.hivemq.com-v1.yml
          ./openapi2jsonschema.py charts/hivemq-operator/crds/hivemq-cluster.yaml
```